sites/mosul/configs/openmrs/initializer_config/ampathforms/F03-mhGAP_Baseline.json

```
@@ -5,7 +5,7 @@
     "published": true,
     "uuid": "",
     "processor": "EncounterFormProcessor",
-    "encounter": "Consultation",
+    "encounter": "MH Consultation",
```

@michaelbontyes shouldn't be use full names for this? 
```suggestion
    "encounter": "Mental Health Consultation",
```